live-examples/js-examples/string/meta.json

```diff
@@ -1,5 +1,19 @@
 {
     "pages": {
+        "stringIterator": {
```

Comment:
It looks like meta.json hasn't been updated - it included entries from another PR, but not one for `raw`